group_vars/all.yml

```diff
@@ -290,3 +290,16 @@ osds_per_device: 1
 #overcloud_timeout:
 foreman_url: https://foreman.example.com
 manila_enabled: false
+
+# TLS-Everywhere has been tested on OSP 17.
+tls_everywhere: false
+# vars for TLS-Everywhere deployment
+# The host specified as idm_host_fqdn must not be present
+# in instackenv.json, and must be in the same allocation
+# that the OSP deployment will be in.
+idm_host_fqdn:
+ipa_domain:
+ipa_realm:
+ipa_password:
+ipa_ntp_server:
```

Comment:
can you minimize the variables to be set by users. Instead set default values

Reply:
@sanjaychari you can do something like this for setting default values for IDM vars
https://github.com/redhat-performance/jetpack/pull/153/files#diff-0c5e8f9ce36621141ff343acf17436f40ed11c0e0e931b70b0ed28d1dabe6769R1-R40

Reply:
I can set default values for ipa_domain and ipa_realm.
But the user still has to set idm_host_fqdn(because they have to choose which host they want to use for the IdM server in their allocation), ipa_password, ipa_ntp_server and rhos_release_rpm_link since these are sensitive variables that cannot be set to default values.